kernel/main.c

```diff
@@ -340,6 +353,52 @@ async_init(void)
       panic("Unable to create a kthread for do_async_init()");
 }
 
+static void
+setup_uefi_runtime_services() {
```

Comment:
Coding style:
1. In C, functions that don't accept parameters are declared like `foo(void)`. If you skip the `void`, it means that any number of params can be passed. That's different from C++.
2. The '{' goes on the new line.